demisto_sdk/commands/create_id_set/create_id_set.py

```diff
@@ -14,6 +21,54 @@ def __init__(self, output: str = '', input: str = '', print_logs: bool = True):
         self.output = output
         self.input = input
         self.print_logs = print_logs
+        self.id_set = OrderedDict()  # type: ignore
 
     def create_id_set(self):
-        return re_create_id_set(id_set_path=self.output, pack_to_create=self.input, print_logs=self.print_logs)
+        self.id_set = re_create_id_set(id_set_path=self.output, pack_to_create=self.input, print_logs=self.print_logs)
+        self.modify_id_set_command_to_integration_of_playbook()
+        return self.id_set
+
+    def modify_id_set_command_to_integration_of_playbook(self):
```

Comment:
WDYT? could be a different name too
```suggestion
    def add_command_to_implementing_integrations_mapping(self):
```